### ISSUE TITLE
Skeleton - screen reader text should be updated

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11401]
**Describe the enhancement or change**
To follow the a11y docs for the component, namely the [when to apply screen reader text](https://www.patternfly.org/components/skeleton/accessibility#when-to-apply-screen-reader-text) content, we should update examples so only 1 skeleton in each example has screen reader text. Additionally, we should omit any sort of percentage from the text and rely on something slightly more generic, e.g. "Loading default content", "Loading percentage height content" and so on.

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**

**Comment by Mash707** [comment]
Hi @thatblindgeye I would like to work on this.
